spotbugs/src/main/java/edu/umd/cs/findbugs/ba/jsr305/TypeQualifierValue.java

```diff
@@ -148,15 +148,6 @@ private TypeQualifierValue(ClassDescriptor typeQualifier, @CheckForNull Object v
                 try {
                     Global.getAnalysisCache().getClassAnalysis(ClassData.class, checkerName);
 
-                    // found it.
```

Comment:
Maybe the maintainers can tell if we can part with the functionality provided by the ValidationSecurityManager. Does it solve an issue you have?

Reply:
🤔 I'm not so sure about why we need to set a security manager here. The comment on 4a9eaf430f5dcbca923026e05b603bda2930368a gave no hint.

At least existing test cases got no effect from this deletion. It also has already been disabled when we run SpotBugs as an IDE plugin. So I think it's OK to remove it. 

😊 @spotbugs/core-devs please share your thought if you have.

BTW SpotBugs usually uses system property instead of environment variables, to switch behavior. Just like the `DEBUG_CLASSLOADING` variable in this class. So if we need a feature toggle, better to use a system property.